recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ requirements:
     - matplotlib-base >=1.5.1
     - seaborn
     - netcdf4 >=1.0
-    - biopython >=1.71
+    - biopython >=1.80
```

Comment:
updated to 1.80 as per https://github.com/MDAnalysis/mdanalysis/blob/develop/package/setup.py#L597

unfortunately we forgot to update our pyproject.toml... tbd?